### DISCUSSION TITLE
Hi there!

### DISCUSSION THREAD

**Discussion by ghost** [janet-lang/janet#1425] [General]
I was watching _The Good Place_ with someone and I told them that the latter three seasons are a waste of time. I wanted to know if that was a fair assessment so I binged the entire series in three days. I must admit, I was wrong.

It's true, the first season is watertight in a way that the rest aren't, but the rest do cohere when taken as a whole. Being released one episode at a time really does a show like that dirty.

And even if you absolutely _detest_ the experience of watching the latter three seasons, I say it's worth it just for that **ending**. My _god_, it's incredible. 104% perfect, wrap up the entirety of television, we're done.

And yes, Janet is my favourite character. She's the platonic ideal of everything I aspire to be: kind, enthusiastic, hyperintelligent, omniscient. Also she kind of looks like me which is nice for reasons which some of you might understand.

Can she die, though? I was never clear on that. First rebooting was death, then she came back immediately; then marblising was death, but next we see her she's reconstituted in the Bad Place with no explanation. It's cool if she can't die (and I can forgive the false start for obvious reasons), I just wish that was better explained.

Speaking of death, I feel for her. The only character that doesn't get to die at the end. I know the present is little more than a knowledge horizon for her, but still, eternal existence is a drag. If it really doesn't affect her, I am so jealous. To have a role, be perfect at it, have it be the highest fulfilment your kind can achieve, and maintain all of that forever; that's more than I'd ever dare to dream.

Oh, um, spoilers. And, rambling. Sorry, read it backwards, loop around. Anyways I love that Janet is named Janet and that it's kind of the platonic ideal of scripting languages, in a way. Kind of like if a Lisp was _actually_ designed for symbolic expression of programs without an irrational attachment to first principles or the specific machine that happened to be on hand at the time.

Also, thanks bakpakin. For, not being a wanker. For just doing your thing, and not getting your ego mixed up in it. I wish I could do that.

**Comment by bakpakin** [comment]
Thanks for the thoughts. Like most shows, the Good Place. there are ups and downs, just as with the language.

I try to be realistic with what I can accomplish with Janet, and that necessarily means limiting the scope. It's never going to be perfect, but I do think we can make it good.

**Reply by pepe** [reply]
Amen

**Comment by pepe** [comment]
I agree with and love everything you wrote here, whether for the series, the language, or our lord. <3

[the remark this answers is not in the document]
**Reply by sogaiu** [reply]
Well, one might argue that @ianthehenry is the current [judge](https://github.com/ianthehenry/judge/)...sorry, couldn't resist.
